test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
Do these really require objc interop or does this all work on Linux with swift-corelibs-foundation too?